src/include/sof/lib/dai.h

```diff
@@ -470,6 +471,14 @@ static inline uint32_t dai_get_init_delay_ms(struct dai *dai)
 	return 0;
 }
 
+static inline int dai_get_fifo_depth(struct dai *dai, int direction)
```

Comment:
this one returns `int` which seems perfectly reasonable for a "number of elements or negative error"